packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -348,6 +379,8 @@ class _BottomNavigationTile extends StatelessWidget {
     // (which is an integer) by a large number.
     int size;
 
+    final double selectedFontSize = selectedLabelStyle.fontSize;
```

Comment:
It actually is used just below here, it's just that `selectedFontSIze` used to be a field. Since it's no longer a field, I created this local variable with the same name.